### DISCUSSION TITLE
Pondering a name change

### DISCUSSION THREAD

**Discussion by Uncommon** [Uncommon/Xit#466] [General]
A problem with Xit as a name is that the intended pronunciation is "exit", but people tend to read it as "zit". A product name is like a joke - if you have to explain it, it doesn't work.

Two of my favorite ideas so far are:

* **Frigate**: it sort of has "git" in the name, and the shape of a [frigate bird](https://en.wikipedia.org/wiki/Frigatebird) in flight could make for a cool icon. But does it sound too much like a curse word? And if so, is that good or bad?
* **Genet**: has two meanings. One is another word for clonal colony, such as an aspen grove. This is very appropriate for a Git-related tool. The other is an [African cat-like animal](https://en.wikipedia.org/wiki/Genet_(animal)), which again could make a cool icon.

Any thoughts from the community at large would be appreciated.

**Comment by JanX2** [comment]
Funny thing is, I never thought much about the name until this issue popped up in my notifications. Then I immediately thought “I guess something like a pimple isn’t a good image to have when thinking of an app.“ your issue  description confirmed my suspicion.

I think both options are equally good, but not great. Very creative though! 

These aren’t great, but may inspire some better ones:
Riffing on the “Exit” theme:
Chute
Door
…

Extending “Exit”:
Xit Strategy
Xitus (probably too much dark humor)

**Reply by Uncommon** [reply]
I actually don't feel attached to the "exit" theme. It started as a variant of GitX but with the X in a different place, so it ended up sounding like "exit" as sort of a random pun.
